lib/constants.js

```diff
@@ -79,7 +80,7 @@ const ADD_COMPONENT = {
     }
 };
 
-if (tarsUtils.isTarsInited().inited && tarsUtils.getTarsProjectVersion() >= '1.8.0') {
+if (tarsUtils.isTarsInited().inited && semver.cmp(tarsUtils.getTarsProjectVersion(), '>=', '1.8.0')) {
```

Comment:
You have fixed one ugly bug) tars add-component is not working now, cause with code is not executed in 1.10.x. Good job!